packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -131,6 +135,11 @@ export interface IEditableTextState {
 const BUFFER_WIDTH_DEFAULT = 5;
 const BUFFER_WIDTH_IE = 30;
 
+export enum TriggerSource {
```

Comment:
since this is being exported from the package root as public API, I think it needs to have a more explicit name like `InputConfirmationKind` (similar to `PopoverInteractionKind`). or you could take a more conservative approach and just inline the string literal type, avoiding adding to the public API (`onConfirm?(value: string, triggerSource: "enter" | "blur"): void`)